content/id/docs/tutorials/kubernetes-basics/expose/expose-intro.html

```diff
@@ -0,0 +1,109 @@
+				<li><i>ExternalName</i> - Mengekpos Service menggunakan nama sesuai keinginan (ditentukan oleh <code>externalName</code> dalam <i>spec</i>) dengan mengembalikan catatan CNAME dengan nama tersebut. Tidak menggunakan proksi. <i>Type</i> ini membutuhkan <code>kube-dns</code> versi v1.7 atau lebih tinggi.</li>
+			</ul>
+			<p>Informasi lebih tentang <i>type</i> Service berbeda dapat ditemukan di tutorial <a href="/docs/tutorials/services/source-ip/">Menggunakan IP pengirim</a>. Lihat juga <a href="/id/docs/concepts/services-networking/connect-applications-service">Menghubungkan aplikasi dengan Service</a>.</p>
+			<p>Selain itu, catat bahwa ada kasus penggunaan Service yang tidak mendefinisikan <code>selector</code> di <i>spec</i>. Sebuah Service yang dibuat tanpa <code>selector</code> juga tidak akan membuat objek Endpoint yang sesuai. Ini mengizinkan pengguna untuk memetakan Service secara manual ke <i>endpoint</i> spesifik. Kemungkinan lain kenapa dimungkinan tanpa <i>selector</i> adalah kamu dengan tegas menggunakan <code>type: ExternalName</code>.</p>
+			</div>
+			<div class="col-md-4">
+				<div class="content__box content__box_lined">
+					<h3>Ringkasan</h3>
+					<ul>
+						<li>Mengekspos Pod ke kunjungan eksternal</li>
+						<li><i>Load balancing</i> kunjungan melintasi beberapa Pod</li>
+						<li>Menggunakan label</li>
+					</ul>
+				</div>
+				<div class="content__box content__box_fill">
+						<p><i>Service Kubernetes adalah lapisan abstraksi yang mendefiniskan kumpulan Pod secara logis dan mengijinkan paparan kunjungan eksternal, <i>load balancing</i>, dan <i>service discovery</i> untuk Pod-Pod tersebut.</i></p>
+				</div>
+			</div>
+		</div>
+		<br>
+
+		<div class="row">
+			<div class="col-md-8">
+				<h3>Service dan Label</h3>
+			</div>
+		</div>
+
+		<div class="row">
+			<div class="col-md-8">
+				<p><img src="/docs/tutorials/kubernetes-basics/public/images/module_04_services.svg" width="150%" height="150%"></p>
+			</div>
+		</div>
+
+		<div class="row">
+			<div class="col-md-8">
+				<p>Service mengarahkan kunjungan melintasi satu set Pod. Service merupakan abstraksi yang mengizinkan Pod-Pod untuk mati dan replikasi dalam Kubernetes tanpa memengaruhi aplikasimu. Menemukan dan <i>routing</i> antara Pod dependen (seperti komponen <i>frontend</i> dan <i>backend</i>) yang ditangani oleh Service Kubernetes.</p>
+				<p>Service mencocokan satu set Pod menggunakan <a href="/id/docs/concepts/overview/working-with-objects/labels/">label dan selektor</a>, mengelompokan primitif yang mengizinkan operasi logika pada objek-objek dalam Kubernetes. Label adalah pasangan <i>key/value</i> yang melekat pada objek-objek dan dapat digunakan untuk beberapa tujuan:</p>
+				<ul>
+					<li>Menandai objek-objek untuk <i>development</i>, <i>test</i>, dan <i>production</i></li>
```

Comment:
how about adding "lingkungan" to emphasize on environment?
```suggestion
					<li>Menandai objek-objek untuk lingkungan <i>development</i>, <i>test</i>, dan <i>production</i></li>
```